package.json

```diff
@@ -20,7 +20,7 @@
         "test:bundle": "jest --env=jsdom e2e-tests/bundle/**",
         "test:e2e": "jest -c e2e-tests/jest.config.js",
         "test:e2e:start": "serve",
-        "typecheck": "tsc --noEmit && tsc types/*.ts --noEmit",
```

Comment:
The `types` folder is now included in the tsconfig: https://github.com/paypal/paypal-js/blob/main/tsconfig.json#L15